pkg/controller/handler.go

```diff
@@ -12,41 +12,85 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
+const SAS_PREFIX = "k8scc01covidacr.azurecr.io/sas:"
+const FEATURES_LABEL = "state.aaw.statcan.gc.ca/employee-only-features"
+const RB_LABEL = "state.aaw.statcan.gc.ca/non-employee-user"
+
 func sasImage(pod *corev1.Pod) bool {
-	image := pod.Spec.Containers[0].Image
-	sasImage := strings.HasPrefix(image, "k8scc01covidacr.azurecr.io/sas:")
-	return sasImage
+
+	for _, container := range pod.Spec.Containers {
+		sasImage := strings.HasPrefix(container.Image, SAS_PREFIX)
+		if sasImage {
+			return true
+		}
+	}
+
+	return false
 }
 
-func (c *Controller) handleProfile(profile *v1.Profile) error {
-	ctx := context.Background()
+func isEmployee(email string) bool {
+
+	employeeDomains := [2]string{"cloud.statcan.ca", "statcan.gc.ca"} // Intialized with values
```

Comment:
Consider making `employeeDomains` a global variable (because we might have a new domain in the future and want to update in only one place).